src/foam/nanos/controller/ApplicationController.js

```diff
@@ -853,7 +853,7 @@ foam.CLASS({
       if ( hash && hash != 'null' /* How does it even get set to null? */ && hash != this.currentMenu?.id ) {
         this.window.onpopstate();
       } else {
-        this.pushMenu('');
+        this.pushMenu('', true);
```

Comment:
I know this would fix it but this isnt the real issue, real issue should be fixed with kgr's memento updates

Reply:
is that coming soon? should we merge for now and revert later?

Reply:
Yes, let's merge now, then revert.